src/appendices/qmri.md

```diff
@@ -303,7 +303,7 @@ and a guide for using macros can be found at
 | -------------------- | ---------------------------------------------------------------------------------------------------- |
 | TB1DAM               | `FlipAngle`                                                                                          |
 | TB1EPI               | `EchoTime`, `FlipAngle`, `TotalReadoutTime`, `MixingTime`                                            |
-| TB1AFI               | `RepetitionTime`                                                                                     |
+| TB1AFI               | `RepetitionTimeExcitation`                                                                           |
```

Comment:
@agahkarakuzu I believe this is what TB1AFI should have said, but I want to run it by you.